pset-iterators.js

```diff
@@ -138,6 +186,13 @@ console.log('Problem 5:')
 
 // Add your code below this line
 
+
+const livingPresidents = presidents.filter( president =>{
+  return president.death_year === null
+})
+
+console.log(livingPresidents)
```

Comment:
@thetripler Good work here using the filter method to return presidents who are still alive today.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)